<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>358</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_pattern">
      <attribute name="title">
       <string>Pattern</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,10,0">
       <item>
        <widget class="AppliedLoadWidget" name="appliedLoadWidget" native="true"/>
       </item>
       <item>
        <widget class="TableWidget" name="tableWidget" native="true"/>
       </item>
       <item>
        <widget class="FastenerWidget" name="fastenerWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_solver">
      <attribute name="title">
       <string>Solver</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,10">
       <item>
        <widget class="SolverWidget" name="solverWidget" native="true"/>
       </item>
       <item>
        <widget class="ResultWidget" name="resultWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AppliedLoadWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/AppliedLoadWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FastenerWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/FastenerWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/TableWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SolverWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/SolverWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/ResultWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
